### PR TITLE
Initial commit of example analyzers

### PR DESCRIPTION
**Pull request by notslow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
...
+cd dragonfly-analyzers
+sh analyzer-install.sh filter
+```
+
+### Step 6: Create config.lua
+
+The `config.lua` file determines the analyzers applied to each event type and the order in which they are applied. There are example config files located in the test directories.  We have included example configurations to add a priority score to IDS alerts and run a DGA detector. They are included in the `dragonfly-analyzers/config` directory. For example, to use the priority score configuration, run the following commands: 
+```
+cd dragonfly-analyzers
+cp event-triage-config.lua /usr/local/dragonfly-mle/config/config.lua
```
Comment (RudeDude) [inline README.md]: either "cp config/event-triage-config.lua ..." etc, or change the previous line to cd dragonfly-analyzers/config

File: analyzer-install.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## If you prefer to use rsync, which is not installed by default on OPNids
+# echo "rsync -vvu $rsync_dirs $dragonfly_root/$destination"
+# rsync -vvu $rsync_dirs $dragonfly_root/$destination
+
+# Perform the copy, using "no-clobber" and "verbose" options. The script won't copy over
+# any files that have been updated in the analyzer directory
+# Note: You must copy all config files manually
+echo "cp -nv $rsync_dirs $dragonfly_root/$destination"
+cp -nv $rsync_dirs $dragonfly_root/$destination
```
Comment (RudeDude) [inline analyzer-install.sh]: Maybe use gnu `install` instead of cp here

Comment (RudeDude) [reply analyzer-install.sh]: I'll test this updated version tonight or tomorrow.

File: README.md [hunk 3]
```diff
@@ -1 +1,160 @@
...
+./bin/dragonfly-mle
+```
+If you are using OPNids, the MLE can also be restarted from the GUI. (https://docs.opnids.io/manual/gui.html)
+
+### Step 8: Check the Output of the MLE
+
+To check the output of the MLE, look at the `eve-mle.json` file.
+
+```
+wc -l /var/log/dragonfly-mle/eve-mle.json
```
Comment (RudeDude) [inline README.md]: I think this log is actually named `/var/log/dragonfly-mle/eve-mle.log` not dot json.

**Comment by RudeDude** [comment]
Ok, I was able to walk through the readme (on standalone, not OPNids) successfully! I also ran the included Docker file as the test suite with no problems (although the DGA cache test was failing due to that race condition you mention).

Last couple items / questions:

- The random forest definition files are named `*.ml` which is confusing the GitHub code type counter. Any chance of changing the file extension (and code that then loads them)? Super low priority request.
- The ip-util/IP2LOCATION-LITE-DB1.CSV  file seems to be under the CC sharealike 4.0 license... do we need to note that somewhere in the folder or parent folder?
- The entire IP-UTIL folder is 60 MB, is there a chance we could not have that all checked into Git? Like replace it with a series of wget requests that are scripted as part of the make / install steps?

**Comment by RudeDude** [comment]
One other "test" I did was to run the bats tests from within the base MLE docker I had setup for testing and they ran mostly fine but had trouble with redis stopping / not-restarting for IP ASN lookup and 'time anomaly'.

File: README.md (hunk 3 above)
Comment (notslow) [reply README.md]: Agreed. Now fixed.

**Comment by notslow** [comment]
@RudeDude Check out the latest changes to see if your comments have been addressed.

**Comment by RudeDude** [comment]
Install doesn't do the /www folder with its json files

**Comment by RudeDude** [comment]
Everything checks out. The /www folder location being hardcoded is an MLE bug that I can work on but this all loads up and runs nicely! I suggest I do a merge+squash onto master? Also the MLE tip-of-master has the p/rs merged already so that will match. @notslow

**Pull request merged**